pkg/eval/str/str_test.go

```diff
@@ -31,6 +31,14 @@ func TestStr(t *testing.T) {
 		That(`str:equal-fold abc ABC`).Puts(true),
 		That(`str:equal-fold abc A`).Puts(false),
 
+		That(`str:from-codepoints 0x61`).Puts("a"),
+		That(`str:from-codepoints 0x4f60 0x597d`).Puts("你好"),
+		That(`str:from-codepoints -1`).ThrowsAny(),
```

Comment:
Don't use ThrowAny; test the exact exception thrown.

Reply:
I wave my magic wand and mutate the existing 42 uses of `ThrowsAny()` to instead check for a specific exception. :smile: On a more serious note I'll point out that the "throw" related test methods in *pkg/eval/testutils.go* would benefit from some documentation. If nothing else that `ThrowsAny()` is deprecated.